test-resources/src/functions/common/logging/LogMessage.ts

```diff
@@ -97,6 +101,20 @@ export class LogMessage implements LogAttributes {
     "The sessionId path parameter is missing or invalid",
   );
 
+  // DynamoDB adapter
+  static readonly PUT_ITEM_ATTEMPT = new LogMessage(
+    "TEST_RESOURCES_PUT_ITEM_ATTEMPT",
+    "Attempting to put an item into DynamoDB",
+  );
+  static readonly PUT_ITEM_UNEXPECTED_FAILURE = new LogMessage(
+    "TEST_RESOURCES_PUT_ITEM_UNEXPECTED_FAILURE",
+    "Failure occurred while sending item command to DynamoDB",
```

Comment:
nitpick: 'sending item command' makes it sound something different to put item command

```suggestion
    "Failure occurred in put item command to DynamoDB",
```

Or something like that?

Reply:
Updated